PaddleAudio/examples/dcase2021_task1b/README.md

```diff
@@ -0,0 +1,117 @@
+# DCASE challenge 2021 task1b, Audio-visual scene classification
+
+In this example, we give a much <b>stronger</b> baseline for the [task-1b of DCASE challenge](http://dcase.community/challenge2021/task-acoustic-scene-classification#subtask-b).
+
+## Introduction
+
+The task of audio-visual scene classification is to combine audio stream and video stream in order to do scene classification.  Different from sound event classification, scene classification is more complex as different combination of sound events can occur in the same scene. Therefore, scene classification is more prune to over-fitting. In this example, we utilize our pre-trained resent50 in the [audioset training example](../audioset_training) for audio-only classification. For video-only track, we use recent advance in contrastive language-image pre-training, aka CLIP\[5\] to do zero-shot/linear-probe image scene classification, obtained 0.772 and 0.88  overall classification accuracy respectively. .
+
+Finally, we combined both audio and visual stream in later-fusion manner and obtained 0.925 overall accuracy, which largely over-takes the performance in the [ICASSP paper](https://github.com/shanwangshan/TAU-urban-audio-visual-scenes)\[1\].
+
+## Demo structure
+- model.py: the modified resent models, support both audio-visual and audio-only tasks
+- train.py: for training the model
+- evaluate.py: for evaluation for audio-visual and audio-only
+- dataset.py: read mel features from h5
+- utils.py: some utility functions, e.g., mixup
+- video_only_lp.py: linear-probe using clip for video-only task
+- zs_evaluate.py: zero-shot evaluation using clip for video-only task
+- wav2mel.py: convert wav to mel and store in h5, for faster training
+- asserts/: all the necessary files/data/config
+
+
+## Dataset
+
+Data was recorded in the following 10 scenes:
+- Airport - airport
+- Indoor shopping mall - shopping_mall
+- Metro station - metro_station
+- Pedestrian street - street_pedestrian
+- Public square - public_square
+- Street with medium level of traffic - street_traffic
+- Travelling by a tram - tram
+- Travelling by a bus - bus
+- Travelling by an underground metro - metro
+- Urban park - park
+
+### Audio-only
+
+For audio-only scene classification, we extract the same feature as described in  [audioset training example](../audioset_training) and use the same network, namely resent50, except that we add extra fc-layers and one final classification head for the 10-way classification. We also add dropout to reduce over-fitting. See [model.py](model.py) for details.
+
+Training audio-only network:
+``` sh
+python train.py --device=0 --task_type=audio_only --config='./assets/config.yaml'
+```
+
+To evaluate using our pre-trained weights, try
+``` sh
+python evaluate.py --device=0 --task_type=audio_only --config='./assets/config.yaml'
```

Comment:
--device=gpu
--device=cpu
--device=xpu (for ASIC chips)
Please update all the --device usage to correct one.